DrinkingPub/Order.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Xml.Linq;
+
+namespace Vsite.Oom.DrinkingPub
+{
+    public class Order
+    {
+        private Dictionary<Item, int> orderItems;
+
+        public Order()
+        {
+            orderItems = new Dictionary<Item, int>();
+        }
+
+        public Dictionary<Item, int> OrderItems
```

Comment:
Ovo svojstvo (property) vraća referencu na interni objekt `orderItems`, pa preko tog svojstva možete mijenjati sadržaj rječnika. To možete provjeriti npr. tako da u metodi `Table.TotalToPay`, iznad unutarnje `foreach` petlje dodate naredbu:
`order.OrderItems.Add(new Item("Vinjak", 5), 3);`
Da spriječite nekontrolirane promjene, definirajte svojstvo na sljedeći način:
`public IEnumerable<KeyValuePair<Item, int>> OrderItems => orderItems.AsEnumerable();`